package.json

```diff
@@ -74,6 +74,7 @@
         "@netgen/javascript-cookie-control": "^0.0.8",
         "@popperjs/core": "^2.11.6",
         "bootstrap": "^5.3.3",
+        "flatpickr": "^4.6.13",
```

Comment:
Again wrong rebase ? :)

Reply:
Yeah...I don't know what happened, but I had to add it again